tools/swift-demangle/swift-demangle.cpp

```diff
@@ -14,6 +14,10 @@
 //
 //===----------------------------------------------------------------------===//
 
+#if defined(__FreeBSD__)
+#define _WITH_GETLINE
+#endif
```

Comment:
Changes to this file LGTM, feel free to submit as a separate PR if you would like them to be merged immediately.